### ISSUE TITLE
Intrigue

### ISSUE THREAD

**Issue by gabrielloeb** [tisimst/hersheyfonts#2]
This is extremely exciting, did this project come to fruition? I'd love to employ the OTFs!

**Comment by ThomasRettig** [comment]
@gabrielloeb apparently you can find them here: https://github.com/kamalmostafa/hershey-fonts :)

**Comment by tisimst** [comment]
This project did come to fruition, in fact, back when I uploaded the readme. I just never got around to uploading the font files. Life ended up going a very different way shortly thereafter. I'm still not sure how to manage the upload of over 1200 font files. If you want to go with the files at kamalmostafa's repository, then have fun. They're the starting point, but not really what you're looking for. I'm not opposed to uploading the font files, I'm just super busy with my day job.

**Comment by gabrielloeb** [comment]
@tisimst, if you remain open to uploading I’d be happy to try to comb through and organize the output for that purpose. It sounds like this could be a pretty incredible resource for those interested in the Hershey types, and well worth the trouble.

**Comment by ThomasRettig** [comment]
> This project did come to fruition, in fact, back when I uploaded the readme. I just never got around to uploading the font files. Life ended up going a very different way shortly thereafter. I’m still not sure how to manage the upload of over 1200 font files. If you want to go with the files at kamalmostafa’s repository, then have fun. They’re the starting point, but not really what you’re looking for. I’m not opposed to uploading the font files, I’m just super busy with my day job.

@tisimst… Have you tried “Github Desktop”? Assuming that you already have all your 1200 font files in an existing folder on your computer, you can basically hook it up to Github Desktop, and then immediately publish to Github instantaneously. Perhaps this could get you started: https://docs.github.com/en/desktop :) Or have you tried that out already and found out that there is a larger issue at stake?

**Comment by tisimst** [comment]
> Or have you tried that out already and found out that there is a larger issue at stake?

Yeah, it's called _time_, lol. I'll see if Github Desktop lets me do it reasonably easily. It's a lot of files, though.

**Comment by btcooooper** [comment]
Wow—that's a lot of work! Would Github allow you to post the entire collection as a single zip file?

**Comment by tisimst** [comment]
@btcooooper, that's actually a really good idea. That should be easy enough to do.

**Comment by lehni** [comment]
@tisimst I'm very interested in seeing this as well, and happy to help out!

**Comment by yangcht** [comment]
waiting for the font files!

**Comment by ischmaedecke** [comment]
It would be awesome to have access to these font files!
Does anyone know where I could find true monoline fonts to which I can add stroke to control the thickness? Is this even possible in OTF?

THANKS!

**Comment by yangcht** [comment]
> 
Hey @ischmaedecke, you may check here

https://github.com/yangcht/Hershey_font_TTF

**Comment by ischmaedecke** [comment]
> 

hi @yangcht , thanks for sharing. I think those are "faux" monoline fonts, as in: the shapes we see are actual thin rectangles and not derived from a single vector stroke.

**Comment by hattesen** [comment]
> It would be awesome to have access to these font files! Does anyone know where I could find true monoline fonts to which I can add stroke to control the thickness? Is this even possible in OTF?
> 
> THANKS!

I think the closest thing will be these two repositories of Hershey fonts in OTF and TTF format
* https://github.com/scruss/AVHershey-OTF
* https://github.com/Dener-Silva/Hershey-TTF

Late to the party, but someone may benefit.

But no, you will not be able to "add stroke to control the thickness" which wouldn't be possible in either OTF or TTF. Unless, of course, you use a program like [Inkscape](https://inkscape.org/) to convert text into paths using `Path > Union`, in which case you could add stroke width.
